### ISSUE TITLE
code could be cleaner

### ISSUE THREAD

**Issue by freewheel70** [12138mICHAEL1111/redux-demo#1]
https://github.com/12138mICHAEL1111/redux-demo/blob/74179e1162c6b2637775992bff88b20b0e774092/src/reducers/reducer.js#L5

why not
<pre>
return [...state, action.foodName]
</pre>

**Comment by 12138mICHAEL1111** [comment]
Yes, it's a better way of change the state. By the way, I've learned and written some basic original code of redux components including connect, dispatch, store etc. If you are interested on it, please look at https://github.com/12138mICHAEL1111/redux-origin
